repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -473,6 +477,32 @@ private static void initProcess(Launcher launcher) {
         listener.runListenerServer();
     }
 
+
+    /* Entry point as Jenkins plugin - skip JSAP */
+    public void jenkinsMain(int buildId) {
```

Comment:
Another design option is to add a new class "JenkinsLauncher extends Launcher". Better separation of responsibilities?

Reply:
Yes, That's true :) . It's fixed in the latest commits .